.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,116 @@
+name: ✨ Feature Request
+description: Suggest an idea for improvement or a new feature
+title: "[Feature]: <Write a concise title here>"
+labels: ["feature"]
+body:
+    - type: checkboxes
+      id: no-duplicate-issues
+      attributes:
+          label: Is there an existing issue for this?
+          description:
+              Please check to see whether the issue you want to create has already been reported.
+          options:
+              - label:
+                    I have searched the existing
+                    [issues](https://github.com/trvswgnr/sloth-pipe/issues)
+                required: true
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to suggest a feature for this project! We are grateful ❤
+    - type: input
+      id: contact
+      attributes:
+          label: Contact Details
+          description: How can we get in touch with you if we need more info?
+          placeholder: ex. email@example.com
+      validations:
+          required: false
+    - type: textarea
+      id: issue-description
+      attributes:
+          label: Description of your feature request
+          description: A clear and concise description of the feature you are requesting.
+      validations:
+          required: true
+    - type: textarea
+      id: motivation
+      attributes:
+          label: Motivation
+          description:
+              Explain why this feature would be beneficial and how it addresses a specific need or
+              problem.
+      validations:
+          required: true
+    - type: textarea
+      id: context
+      attributes:
+          label: Additional Context
+          description: Add any additional context or screenshots about the feature request here.
+      validations:
+          required: false
+    - type: textarea
+      id: implementation
+      attributes:
+          label: Possible Implementation
+          description:
+              If you have any thoughts on how this feature could be implemented or suggestions on
+              its design, feel free to share.
+      validations:
+          required: false
+    - type: textarea
+      id: related-problem
+      attributes:
+          label: Is your feature request related to a problem? Please describe
+          description: A clear and concise description of what the problem is.
+          placeholder: Ex. I'm always frustrated when [...]
+      validations:
+          required: false
+    - type: textarea
+      id: additional-info
+      attributes:
+          label: Additional Information
+          description:
+              Add any other information or context that might be relevant to the feature request.
+      validations:
+          required: false
```

Comment:
I think that it should be more than enough with the fields of motivation and additional context tho.